hack/aws-acceptance-test-cleanup/main.go

```diff
@@ -68,6 +69,94 @@ func realMain(ctx context.Context) error {
 	eksClient := eks.New(clientSession, awsCfg)
 	ec2Client := ec2.New(clientSession, awsCfg)
 	elbClient := elb.New(clientSession, awsCfg)
+	iamClient := iam.New(clientSession, awsCfg)
+
+	// Find OIDC providers to delete.
+	oidcProvidersOutput, err := iamClient.ListOpenIDConnectProvidersWithContext(ctx, &iam.ListOpenIDConnectProvidersInput{})
```

Comment:
ListOpenIDConnectProvidersWithContext can return a nil value, so maybe we should check for that here since you access a member later

Reply:
also should check the error

Reply:
nice catch! thanks!